arch/arm64/boot/dts/freescale/crocodile.dtsi

```diff
@@ -139,94 +144,104 @@
 		interrupts = <3 IRQ_TYPE_LEVEL_LOW>;
 
 		regulators {
+			// VDD_SOC_0V8
 			buck1_reg: BUCK1 {
 				regulator-name = "BUCK1";
-				regulator-min-microvolt = <600000>;
-				regulator-max-microvolt = <2187500>;
+				regulator-min-microvolt = <780000>;
+				regulator-max-microvolt = <900000>;
 				regulator-boot-on;
 				regulator-always-on;
 				regulator-ramp-delay = <3125>;
-				nxp,dvs-run-voltage = <820000>;
 				nxp,dvs-standby-voltage = <800000>;
 			};
 
+			// VDD_ARM_0V9
 			buck2_reg: BUCK2 {
 				regulator-name = "BUCK2";
-				regulator-min-microvolt = <600000>;
-				regulator-max-microvolt = <2187500>;
+				regulator-min-microvolt = <805000>;
+				regulator-max-microvolt = <1050000>;
```

Comment:
Why all these ranges?

Reply:
good point made in out-of-band discussion: set the ranges down to the actually desired values -> see PR #113